docs/angular-testing-library/examples.mdx

```diff
@@ -115,7 +115,7 @@ describe('Counter', () => {
 })
 ```
 
-To override the an import of a standalone component for your component test, you can use the [`ɵcomponentImports` property](api.mdx#ɵcomponentImports]).
+To override the an import of a standalone component for your component test, you can use the [`componentImports` property](api.mdx#componentImports).
```

Comment:
👋 I did a drive-by and spotted a typo that wasn't caused by this commit.
Thanks Santosh!

```suggestion
To override an import of a standalone component for your component test, you can use the [`componentImports` property](api.mdx#componentImports).
```

Reply:
Done